src/box/engine.h

```diff
@@ -296,6 +314,13 @@ struct engine {
 	struct rlist link;
 };
 
+/** True if `engine' supports cross-engine transactions. */
+static inline bool
+engine_supports_cross_engine_tx(const struct engine *engine)
+{
+	return engine->flags & ENGINE_SUPPORTS_CROSS_ENGINE_TX;
+}
+
```

Comment:
Nit: We don't like helper functions for other engine flags so I wouldn't introduce a flag for this one, either. You could use a local variable to avoid long lines.

Reply:
Removed.